process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+We follow the OKR process, see the [2019/2020 Delivery Plan](https://wellcomecloud.sharepoint.com/sites/wc2/DE/RoadMaps/Delivery%20Plans/Delivery%20Plan%202019-20%20Digital%20Engagement.pdf?csf=1&e=WUG2dI&cid=56bdefa5-c5d4-44a1-99e7-96a6975d10fc)
+
+Delivery over process - do not follow these guidelines if they prevent you from delivering value as defined by our objectives.
+
+## Projects
+
+Projects are quarterly iterations of the projects [available in Github](https://github.com/wellcometrust/platform/projects).
+
+Issues are binned or recycled on a quarterly basis.
+
+## Pull Requests (PRs)
+
+All Pull Requests should have in general:
+
+- What is the benefit of this change (and for who)
+  e.g. In order to make searches better for users
+  
+- What is the change functionally
+  e.g. We've ordered searches by date
+  
+- What is the change
+  e.g. By implementing an update to the `ElasticFlarp GargleMap` in `object FleebChunker`
+  
+- Relationships to other changes
+  e.g. Depends on #123, required for #456
+
+For example:
+
+```
+In order to make searches better for users
+We've ordered searches by date
+By implementing an update to the `ElasticFlarp GargleMap` in `object FleebChunker`
+
+Depends on #123, required for #456
+```
+ 
+## Definition of done
```

Comment:
Further to the above, it would be nice to add some documentation of how we write issues/tickets (maybe add an issue template to the repo). So far I've struggled a bit with understanding issues that lack acceptance criteria and/or that seem to require context that isn't documented or linked to them.

Reply:
Intended that the above _is_ that, seems like there is something i'm missing. Probably need to talk this through in person/at retro?